DigitalOcean.API/Clients/DropletActionsClient.cs

```diff
@@ -106,13 +106,13 @@ public class DropletActionsClient : IDropletActionsClient {
         /// A Droplet restoration will rebuild an image using a backup image. The image ID that is passed in must be a backup of
         /// the current Droplet instance. The operation will leave any embedded SSH keys intact.
         /// </summary>
-        public Task<Action> Restore(int dropletId, int imageId) {
+        public Task<Action> Restore(int dropletId, object imageIdOrSlug) {
```

Comment:
API method can allow both imageId integer and slug string. Same as `Requests.Droplet`.`ImageIdOrSlug` or  `IImagesClient.Get(object imageIdOrSlug)` or `IDropletActionsClient.Rebuild(int dropletId, object imageIdOrSlug)`.